_resources/port1.0/group/github-1.0.tcl

```diff
@@ -73,8 +73,11 @@ proc github.setup {gh_author gh_project gh_version {gh_tag_prefix ""} {gh_tag_su
         name                ${github.project}
     }
 
+    if {!([info exists PortInfo(homepage)] && (${PortInfo(homepage)} ne ${github.homepage}))} {
+        homepage            ${github.homepage}
+    }
```

Comment:
For most variables/options, you don't need to go through such contortions to set a default that the portfile can override. For `name` we did because it is a very special and important portfile variable, but for `homepage`, did you try just writing `default homepage {${github.homepage}}`, and leaving it where it was in the portgroup, and without the surrounding `if` statement? If using `default` works, that is the normal and preferable way to accomplish this.

Reply:
works fine! thank you @ryandesign 
commited the new version